recipes/vcflatten/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Any idea why vcflatten appears in this PR?? Or is this an artifact that just I see?